core/src/main/java/org/apache/accumulo/core/logging/TabletLogger.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.logging;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.accumulo.core.dataImpl.KeyExtent;
+import org.apache.accumulo.core.metadata.schema.Ample;
+import org.apache.accumulo.core.tabletserver.log.LogEntry;
+import org.apache.accumulo.core.util.HostAndPort;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// logs messages about a tablets internal state, like its location, set of files, metadata... operations that cut across multiple tablets should go elsewhere
+public class TabletLogger {
+
+  private static final String PREFIX = Logging.PREFIX + "tablet.";
+
+  private static final Logger locLog = LoggerFactory.getLogger(PREFIX + "location");
+  private static final Logger fileLog = LoggerFactory.getLogger(PREFIX + "files");
+  private static final Logger recoveryLog = LoggerFactory.getLogger(PREFIX + "recovery");
+  private static final Logger walsLog = LoggerFactory.getLogger(PREFIX + "wals");
```

Comment:
You went with explicit names, rather than brevity (`location` instead of `loc`), for all of these except `wals`. It'd might be good to try to be consistent here by selecting a more explicit name, like `writeaheadlogs` or compromise with `walogs`.

Reply:
Done in 2fea1a65f31916f327cebf6e7804ad95474cd40b